zone.go

```diff
@@ -231,6 +231,31 @@ type PurgeCacheResponse struct {
 	} `json:"result"`
 }
 
+// ZoneSubscriptionDetails represents all the subscription details for the zone.
+type ZoneSubscriptionDetails struct {
+	ID       string `json:"id,omitempty"`
+	State    string `json:"state,omitempty"`
+	Price    int    `json:"price,omitempty"`
+	Currency string `json:"currency,omitempty"`
+	Zone     Zone   `json:"zone,omitempty"`
+	RatePlan struct {
+		ID                string `json:"id,omitempty"`
+		Name              string `json:"public_name,omitempty"`
```

Comment:
I'm unsure whether it's a good idea to introduce a field that doesn't match the JSON response's key as most of the existing structs follow a 1:1 mapping. Is there a particular reasoning behind this?